index.html

```diff
@@ -94,9 +102,130 @@ <h2 class="position">Software Developer</h2>
       <a href="#"> <i class="fa fa-medium" aria-hidden="true"></i></a>
     </div>
 
-    <!--background shapes-->
-    <div class="shape-model1"></div>
-    <div class="shape-model2"></div>
+   <!--projects section--> 
+
+             <div class="go-to">
+                <img class="arrow-down" src="./arrow-down.svg" alt="">
+                <h4 class="go-work">Work</h4>
+            </div>
+            
+            <h1 class="work-title">MY RECENT WORK</h1>
+              <!--vector line-->
+            <div><hr class="vector-line" /></div>
+
+     <section class="all-work">
+          
+                     
+            <!--first project-->
+            
+            <div id="project1"  class="project-container">
+                <!--background shapes-->
+                <div class="eclipse1"></div>
+
+                <div class="project-image">
+                </div>                
+
+                <h2 class="project-title">Multi-Post Stories</h2>
+                <p class="project-infor" >A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            
+              <ul class="used-tools" >
+                <li class="project-list">css</li>
+                
+                <li class="project-list">html</li>
+                
+                <li class="project-list">bootstrap</li>
+               
+                <li class="project-list">Ruby</li>
+                </ul>
```

Comment:
Please ensure that each section has the margins (top, bottom, left, right) defined in the design.

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/170846841-86c6e98c-8fa9-41a9-8796-111e2636d69a.png) | ![image](https://user-images.githubusercontent.com/10332499/170846836-9778ae72-d15d-4f1a-bf34-d74df8a32782.png)